task/10-katas-1-tasks.js

```diff
@@ -137,7 +245,26 @@ function canDominoesMakeRow(dominoes) {
  * [ 1, 2, 4, 5]          => '1,2,4,5'
  */
 function extractRanges(nums) {
-    throw new Error('Not implemented');
+    let res = "";
+
+    for(let i = 0; i < nums.length; i++){
+        let j = 0;
+
+        if((nums[i + 1] - nums[i] == 1) && (nums[i + 2] - nums[i] == 2)){
+
+            while(nums[i + 1] - nums[i] == 1){
```

Comment:
"operator === is always preferable
sometimes ''==' may cause an error
(check at console :
'0' == 0
and '0' === 0)"